schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/mail/DefaultMailService.java

```diff
@@ -81,25 +81,26 @@ private void initialize() {
     @Override
     public boolean sendToApplicationOwners(String subject, String htmlBody, String textBody) {
         initialize();
-        return send(senderAddress, applicationOwners, subject, htmlBody, textBody);
+        return send(senderAddress, applicationOwners, new InternetAddress[]{}, subject, htmlBody, textBody);
     }
 
     @Override
     public boolean send(String toEmail, String subject, String htmlBody, String textBody) throws Exception {
         initialize();
         InternetAddress[] toAddress = {new InternetAddress(toEmail)};
-        return send(senderAddress, toAddress, subject, htmlBody, textBody);
+        return send(senderAddress, toAddress, applicationOwners, subject, htmlBody, textBody);
```

Comment:
This means we get a copy of EVERY e-mail that is sent...  Good for now for debug, but is more e-mails than before